tests/tck/test_tck.py

```diff
@@ -12,7 +12,7 @@
 
 def test_tck():
     env = Env(decodeResponses=True)
-    cmd = ["./features/", '--tags=-crash', '--tags=-skip']
+    cmd = ["./features/", '--tags=-crash', '--tags=-skip', "--no-capture"]
```

Comment:
Leftover from debugging?

Reply:
no this is so we can print from the behave code and see what printed